docs/assemblies/assembly-record-encryption-filter.adoc

```diff
@@ -6,18 +6,21 @@
 = Record Encryption filter
 
 [role="_abstract"]
-Streams for Apache Kafka Proxy's Record Encryption filter enhances the security of Kafka messages.
+Kroxylicious's Record Encryption filter enhances the security of Kafka messages.
```

Comment:
Do we want to introduce a variable for Kroxylicious to facilitate the production of derivative works?